fastai/vision/augment.py

```diff
@@ -362,13 +362,25 @@ def _grid_sample(x, coords, mode='bilinear', padding_mode='reflection', align_co
     return F.grid_sample(x, coords, mode=mode, padding_mode=padding_mode, align_corners=align_corners)
 
 # Cell
-def affine_grid(theta, size, align_corners=None):
+def affine_grid(
+    theta:Tensor, # A batch of Affine transformation matrices
+    size:tuple, # Output size
+    align_corners:bool=None # `F.grid_sample` align_corners
+):
+    " Generates `TensorFlowField` from a transformation affine matrices `theta`"
     return TensorFlowField(F.affine_grid(theta, size, align_corners=align_corners))
 
 # Internal Cell
 @patch
-def affine_coord(x: TensorImage, mat=None, coord_tfm=None, sz=None, mode='bilinear',
-                 pad_mode=PadMode.Reflection, align_corners=True):
+def affine_coord(x: TensorImage,
+     mat:Tensor=None, # A batch of Affine transformation matrices
+     coord_tfm:callable=None, # Coordinates transformation function
+     sz:(int, tuple)=None, # Output size, duplicated if one value is specified
+     mode:str='bilinear', # `F.grid_sample` mode ( 'bilinear', 'nearest', 'bicubic')
+     pad_mode=PadMode.Reflection, # A `PadMode`
+     align_corners=True # `F.grid_sample` align_corners
+):
+    "Applying affine transformation matrice to `TensorImage`"
```

Comment:
I made some changes for specificity as a suggestion. Please replicate on the other affine patches and `AffineCoordTfm`

```suggestion
def affine_coord(x: TensorImage,
     mat:Tensor=None, # Batch of affine transformation matrices
     coord_tfm:callable=None, # Partial function of composable coordinate transforms
     sz:(int, tuple)=None, # Output size, duplicated if one value is specified
     mode:str='bilinear', # PyTorch `F.grid_sample` interpolation applied to `TensorImage`
     pad_mode=PadMode.Reflection, # Padding applied to `TensorImage`
     align_corners=True #  PyTorch `F.grid_sample` align_corners
) -> TensorImage:
    "Apply affine and coordinate transforms to `TensorImage`"
```